tests/sentry/eventstore/test_models.py

```diff
@@ -439,6 +440,63 @@ def test_get_hashes_gets_hashes_and_variants_if_none_on_event(self):
         assert default_variant.component.values[0].id == "message"
         assert default_variant.component.values[0].values == ["Dogs are great!"]
 
+    def test_has_stacktrace(self):
```

Comment:
Could you please split this into multiple cases?

Can you please pull these tests out of this class? I don't think we need DB support to run these tests and would make them faster.